backend/src/main/java/com/github/redstylzz/backend/model/dto/PaymentDTO.java

```diff
@@ -0,0 +1,17 @@
+package com.github.redstylzz.backend.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+public class PaymentDTO {
+
+    String paymentID;
+    String categoryID;
+    String description;
+    double amount;
```

Comment:
`double` can lead to unexpected results. See: https://www.javacodemonk.com/which-data-type-would-you-choose-for-storing-monetary-values-in-java-6692c78c

BigDecimal is recommended for monetary values